### DISCUSSION TITLE
[꿀팁] yaml 앵커와 별칭

### DISCUSSION THREAD

**Discussion by areum514** [bcaitech1/p4-opt-5-vibrhanium-#28] [꿀팁]
`학교에서 알려주지 않는 17가지 실무 개발 기술이라는 책`의 내용 중 일부입니다. 
링크 첨부합니다. 

앵커와 별칭 파트에서 
객체에 앵커와 별칭 사용하기 부분 보시면 코드 중복되는 부분 해결 할 수 있을 것 같습니다 ;) 

[yaml](https://velog.io/@yjok/YAML#:~:text=%EC%95%B5%EC%BB%A4%EB%8A%94%20%26%20%EB%A1%9C%20%EC%8B%9C%EC%9E%91%ED%95%98%EB%8A%94,%EC%9D%84%20%EC%B0%B8%EC%A1%B0%ED%95%A0%20%EC%88%98%20%EC%9E%88%EB%8B%A4.)

**Comment by olenmg** [comment]
죠아용
